outputs.tf

```
@@ -15,59 +15,59 @@
  */
 
 output "project_id" {
-  value = "${local.project_id}"
+  value = "${module.project-factory.project_id}"
 }
 
 output "project_number" {
-  value = "${local.project_number}"
+  value = "${module.project-factory.project_number}"
 }
 
 output "domain" {
-  value       = "${local.domain}"
+  value       = "${module.project-factory.domain}"
   description = "The organization's domain"
 }
 
 output "group_email" {
-  value       = "${local.gsuite_group ? data.null_data_source.data_final_group_email.outputs["final_group_email"] : ""}"
+  value       = "${module.project-factory.group_email}"
   description = "The email of the created GSuite group with group_name"
```

We should update this description since the main project factory doesn't create the group for you.

```suggestion
  description = "The email of the G Suite group fo the project, if provided"
```